c7n/loader.py

```diff
@@ -167,3 +170,56 @@ def load_file(self):
                 m = r.search(line)
                 if m:
                     self.policies[m.group(2)] = i
+
+
+class DirectoryLoader(PolicyLoader):
```

Comment:
this class feels like it could be split out into its own pr.

Reply:
opened here: https://github.com/cloud-custodian/cloud-custodian/pull/7727